rmf_site_format/src/lib.rs

```diff
@@ -98,6 +95,9 @@ pub use physical_camera::*;
 pub mod point;
 pub use point::*;
 
+pub mod primitive_shape;
+pub use primitive_shape::*;
+
```

Comment:
I moved primitive shapes in the root of `rmf_site_format` from the workcell module. This is necessary since SDFs might contain primitive shapes (usually for collision meshes)